core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -130,6 +131,19 @@ object TestUtils extends Logging {
     f
   }
 
+  /**
+   * Returns true if the elements in both lists are the same irrespective of ordering.
+   */
+  def matchSameElements[T](seq: Seq[T]): Seq[T] = {
+    EasyMock.reportMatcher(new IArgumentMatcher {
```

Comment:
I know this is just moving existing code, but is the comment here accurate?  Looks like this is some specific EasyMock thing rather than a simple assertion on Seq elements which the scaladoc suggests

Reply:
Good point. The documentation is confusing. It looks like I also broke some tests when I tweaked this method. Let me fix that.